src/window/sdl_window.rs

```diff
@@ -179,24 +198,52 @@ impl SdlWindow {
                     }
                 }
                 Event::MouseButtonUp {
-                    x, y, mouse_btn, ..
+                    x,
+                    y,
+                    mouse_btn,
+                    window_id,
+                    ..
                 } => {
+                    if window_id != window_id {
+                        return None;
+                    }
```

Comment:
Isn't this always true? I think one of the variables should be renamed.

Reply:
Fixed an issue with dropping events if they're for other windows and we don't get to them. Also added an example which can contain an infinite amount of windows.